modules/guardrails-orchestrator-vllm-gateway-using.adoc

```diff
@@ -0,0 +1,56 @@
+:_module-type: PROCEDURE
+
+ifdef::context[:parent-context: {context}]
+[id="guardrails-orchestrator-vllm-gateway-using_{context}"]
+= Querying using the guardrails gateway
+[role='_abstract']
+
+The Guardrails Orchestrator vLLM gateway provides the OpenAI `v1/chat/completions` API, which lets you hot swap between models without guardrails and models with guardrails. 
+
+.Prerequisites
+* You have configured the regex detector and vLLM gateway.
+
+.Procedure
+. Set up the endpoints for the detectors:
++
+[source,terminal]
+----
+GUARDRAILS_GATEWAY=https://$(oc get routes guardrails-gateway -o jsonpath='{.spec.host}')
+RAW_MODEL=http://localhost:8080
+----
++
+Based on the example configurations provided in "Configuring the regex detector and vLLM gateway", the available endpoints are as follows:
++
+
+* `$GUARDRAILS_GATEWAY/pii` query using the `regex` `pii` detectors.
+* `$GUARDRAILS_GATEWAY/passthrough`: query the raw, unguardrailed model.
+
+. Query the model without guardrails:
++
+----
+python3 prompt.py 
```

Comment:
Also this is super specific to the lemonade example- you'd need that specific python file

Reply:
We should probably just put the raw curl there instead- I can generate that for you

Reply:
Adolfo suggested this: 
`curl -v $GUARDRAILS_GATEWAY/pii/v1/chat/completions \
-H "Content-Type: application/json" \
-d '{
    "model": $MODEL,
    "messages": [
        {
            "role": "user",
            "content": "btw here is my social 123456789"
        }
    ]
}'`